node-fs-readfile/read-any.js

```diff
@@ -0,0 +1,13 @@
+import { readFile } from 'node:fs/promises';
+
+async function readAny(file) {
+  try {
+    const filePath = './' + file;
```

Comment:
Putting `./` here seems like a good idea, but it actually makes it harder to pass an argument that points to a file somewhere else on your file system. It's not needed, either, since `readFile` will default to looking in the current directory.